modules/s3wrapper/src/s3wrapper/s3_client.py

```diff
@@ -0,0 +1,182 @@
+import re
+from datetime import timedelta
+
+from minio import Minio
+from minio.error import ResponseError
+
+
+class S3Client(object):
+    """ Wrapper around minio
+    """
+    
+    def __init__(self, endpoint, access_key=None, secret_key=None, secure=False):
+        self.__metadata_prefix = "x-amz-meta-"
+        self.client = None
+        try:
+            self.client = Minio(endpoint,
+                access_key=access_key,
+                secret_key=secret_key,
+                secure=secure)
+        except ResponseError as err:
+            print(err)
```

Comment:
we started using the logging module for printing stuff [logging](https://github.com/ITISFoundation/osparc-simcore/blob/8e928ee91058e0954d28513077532bc89de4f36e/services/director/source/director.py#L13)

Reply:
try here ``_LOGGER.exception( ... ) ``